@here/harp-examples/webpack.config.js

```diff
@@ -19,6 +21,15 @@ const harpFontResourcesPath = path.dirname(require.resolve("@here/harp-fontcatal
 const isProduction = process.env.NODE_ENV === "production";
 const harpBundleSuffix = isProduction ? ".min" : "";
 
+themeList = {
```

Comment:
why is this moved to the webpack config? It would make sense if there are multiple files that need it, but it seems a bit strange to move it away from the performance test.

Reply:
b/c the theme list is different for the internal build (i.e. it uses the internal styles).
Please see my internal change as well